pkgs/servers/polaris/default.nix

```
@@ -31,15 +30,7 @@ rustPlatform.buildRustPackage rec {
     '';
   };
 
-  cargoPatches = [
-    (fetchpatch { # https://github.com/agersant/polaris/pull/213
-      name = "bump-time-crate.patch";
-      url = "https://github.com/agersant/polaris/commit/f625c57d203bdd3f2d7fcd99ccce1032f04d9b91.patch";
-      hash = "sha256-ICScYbSv4sCMbfZN2thhZMXGPcDX89xIhZqBJpGOzrY=";
-    })
-  ];
-
-  cargoHash = "sha256-PnNLSL6YIpM6b3+oCh2eNRNPpCKyvnWEW7uNaYTKzAU=";
+  cargoHash = "sha256-Z3AbYtdNAyKT5EuGtCktEg0fxs/gpKdsrttRkxZhLAU";
```

```suggestion
  cargoHash = "sha256-HTqsghjfSjwOaN/ApPFvWVEoquZzE3MYzULkhUOXIWI=";
```

I have tried to build this package, but nix said that `cargoHash` should be `sha256-HTqsghjfSjwOaN/ApPFvWVEoquZzE3MYzULkhUOXIWI=`.

---

managed to reproduce on darwin with `--check`